### DISCUSSION TITLE
Figma - only meant for mobile apps ?

### DISCUSSION THREAD

**Discussion by digger73** [unoplatform/uno#11213] [Q&A] [answered]
I'm in the process of looking into Figma and UNO in combination, for making web and mobile app. After reading the doc that is available I thought I wanted to try it out to get a better understanding. However, now comes my problem as it seems that the Figma template only is made for mobile apps as the only templates available are for mobile.
Am I missing something here ? or is it correct that the Figma templates only are meant for mobile and doesn't cover web application as 
well ? 
If so, maybe this should be mentioned on the Uno Figma page, so people doesn't get their hopes up.

**Comment by sasakrsmanovic** [comment, marked as answer]
Hi @digger73  Thanks for considering Uno Platform and our Figma plugin. You can certainly use our Figma plugin to design non-mobile apps as well. In the beginning, most of our early adopters were looking for Figma-to-Mobile solutions, so our documentation and samples are a bit biased towards that use case. But you can certainly use the same concepts to design a landscape-oriented, WebApp , or a desktop app. We will be coming out with more documentation and samples for web and desktop scenarios in the near future. I'd love to hear what you are thinking of building which could help us craft our roadmap. Please reach out to sasha at platform.uno so we arrange for a quick feedback call. Thanks!

**Reply by digger73** [reply]
I guess the "Uno Platform Material Toolkit" is more or less the "Uno To Do App" except using Material ?
I guess it is just a question about resizing the page template to a desktop size and go from there. 
First of all I'm just looking into if it is a feasible road ahead using the UNO Platform for porting of a SCADA product that is written in WPF. The reasoning for using UNO going forward is that we are a rather small team so I'm hoping that by using UNO we will get some implementation for free between web app and mobile app. 

Out first priority is a web app and secondary the mobile part of the same. I was hoping that the most trivial pages can be made in the web app layout and then they would "automatically" scale down to mobile size without too much changes.
